package.json

```diff
@@ -104,7 +104,7 @@
     "stylelint-processor-styled-components": "~1.10.0",
     "webpack": "~4.43.0",
     "webpack-cli": "~3.3.0",
-    "webpack-dev-server": "~3.10.0",
+    "webpack-dev-server": "^3.11.0",
```

Comment:
use of ~ or ^ does not matter with package-lock. `npm i package` when updating deps rather than manually editing package.json is def the way to go imo.